### ISSUE TITLE
UI：Add Destination page open blank

### ISSUE THREAD

**Issue by hanqian200705** [zinclabs/zincobserve#451]
### Which ZincObserve functionalities are the source of the bug?

alerts

### Is this a regression?

Yes

### Description

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/4269408/228422050-d9b2d60c-a8c4-4f6b-8ad7-7ac6b70f3286.png">

<img width="1305" alt="image" src="https://user-images.githubusercontent.com/4269408/228422172-573fc742-d248-4526-8b6c-a365bf459d7d.png">

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/4269408/228420018-746de631-1184-4990-840f-6cf87e52b2a3.png">

RandomUUID function is not compatible with HTTP.

<img width="762" alt="image" src="https://user-images.githubusercontent.com/4269408/228420520-99554ac8-2173-4542-b424-e29f5f69502c.png">

### Please provide the version you discovered this bug in (check about page for version information)

```true
Version
v0.3.1

Commit Hash
8dd793a2c2b9e1dc30e05d0d6e3123916b800ea8

Build Date
2023-03-25T18:03:06Z
```

**Comment by hengfeiyang** [comment]
refer to: https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

**Comment by omkarK06** [comment]
@hanqian200705 Which browser are you using and what is the version ?

**Comment by hengfeiyang** [comment]
@omkarK06 v0.3.1 and the latest chrome, i tested the reason is the user mentioned. if we visit use HTTP scheme you will got the problem, but you shouldn't use `http://localhost:5080`, you must use something like this: `http://some-domain.com` or `http://192.168.1.1`.
```
Crypto.randomUUID()
Secure context: This feature is available only in secure contexts (HTTPS).
```

**Comment by gaby** [comment]
I ran into the same issue while using the latest Chrome and Edge

**Comment by hengfeiyang** [comment]
fixed in v0.3.2
